scripts/entrypoint.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
```

Comment:
add the wait-for here, then the check if the default user exists and then run the application

Reply:
`yarn run start --prod`